cdk.json

```diff
@@ -24,6 +24,7 @@
     "@aws-cdk/aws-apigateway:authorizerChangeDeploymentLogicalId": true,
     "@aws-cdk/aws-secretsmanager:useAttachedSecretResourcePolicyForSecretTargetAttachments": true,
     "@aws-cdk/aws-apigateway:requestValidatorUniqueId": true,
-    "@aws-cdk/core:includePrefixInUniqueNameGeneration": true
+    "@aws-cdk/core:includePrefixInUniqueNameGeneration": true,
+    "@aws-cdk/customresources:installLatestAwsSdkDefault": false
```

Comment:
This is unrelated to this PR, I just did it at the same time.  This is because CDK was giving an ominous warning about not having this.